model/src/cfd/rollover_v_1_0_0.rs

```diff
@@ -0,0 +1,156 @@
+use crate::olivia;
+use crate::olivia::BitMexPriceEventId;
+use crate::rollover;
+use crate::Cfd;
+use crate::CfdEvent;
+use crate::CompleteFee;
+use crate::Dlc;
+use crate::EventKind;
+use crate::FeeAccount;
+use crate::FundingFee;
+use crate::FundingRate;
+use crate::Position;
+use crate::Role;
+use crate::RolloverParams;
+use crate::TxFeeRate;
+use anyhow::bail;
+use anyhow::Context;
+use anyhow::Result;
+use time::OffsetDateTime;
+
+impl Cfd {
```

Comment:
Is anyone running `rollover_v1.0`? 
Afaik we only introduced it with 0.4.19 which is a buggy version and no one is running it. That being said, we could decide to not support `rollover_v1.0` as we don't expect people to use it?

Reply:
Yes, that's a valid point. I considered it, but I didn't know if we knew for certain that no one was running it. And I wanted to take the opportunity to see if the approach with multiple "versions" of the same protocol crate would work.

I don't mind if we decide to delete  the `v_1_0_0` module now, but I don't want to make the call because it's more work for me now haha.

Reply:
Given that we released `0.5.0` with rollover v1 I suppose this comment is irrelevant because we will need both.